pkgs/by-name/ri/rime-ice/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "rime-ice";
+  version = "2024.09.25";
+
+  src = fetchFromGitHub {
+    owner = "iDvel";
+    repo = "rime-ice";
+    rev = "2024.09.25";
+    hash = "sha256-R2K2LNycXqsUxXvMMq5fcQJUhDnhQMcTlvryMLeiKH0=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/rime-data
+    rm -rf ./others
+    rm -rf ./.github
+    rm squirrel.yaml weasel.yaml
+    cp -r ./* $out/share/rime-data
+  '';
+
+  passthru.updateScript = unstableGitUpdater {
+    branch = "main";
+    hardcodeZeroVersion = true;
+  };
+
+  meta = {
+    description = "Rime 配置：雾凇拼音 | 长期维护的简体词库";
+    homepage = "https://github.com/iDvel/rime-ice";
+    license = lib.licenses.gpl3Only;
```

Comment:
It is unclear whether the license is `gpl3Only` or `gpl3Plus`.  Please contact upstream.

https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206#how-to-determine-license-3

https://www.gnu.org/licenses/identify-licenses-clearly.html